### ISSUE TITLE
Why is it suddenly 'killed'  run train.py?

### ISSUE THREAD

**Issue by INJNainggolan** [hellochick/ICNet-tensorflow#111]
> step 246 	 total loss = 0.767, sub4 = 1.042, sub24 = 0.553, sub124 = 0.238, val_loss: 0.706 (1.229 sec/step)
step 247 	 total loss = 0.775, sub4 = 0.978, sub24 = 0.558, sub124 = 0.254, val_loss: 0.917 (0.760 sec/step)
step 248 	 total loss = 0.996, sub4 = 1.198, sub24 = 0.775, sub124 = 0.353, val_loss: 1.151 (0.849 sec/step)
step 249 	 total loss = 0.765, sub4 = 0.959, sub24 = 0.647, sub124 = 0.211, val_loss: 0.858 (1.241 sec/step)
2019-07-22 02:07:55.482708: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:97] Filling up shuffle buffer (this may take a while): 59 of 500
2019-07-22 02:08:04.897199: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:97] Filling up shuffle buffer (this may take a while): 131 of 500
2019-07-22 02:08:14.928381: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:97] Filling up shuffle buffer (this may take a while): 213 of 500
Killed

Why is it suddenly 'killed' during run train.py?

**Comment by ZhenpengChenCode** [comment]
Hello, I encountered the same problem, did you solve it?
